modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/lewd_gloves.dm

```diff
@@ -26,7 +26,7 @@
 //ball_mittens reinforced
 /obj/item/clothing/gloves/ball_mittens_reinforced //We getting this item by using handcuffs on normal ball mittens
 	name = "reinforced ball mittens"
-	desc = "Do not put these on. Its REALLY hard to put them off... But they looks so comfortable"
+	desc = "Do not put these on. Its REALLY hard to put them off... But they looks so comfortable."
```

Comment:
`Do not put these on, it's REALLY hard to take them off! But they looks so comfortable...`